src/obb_rules/auto_deploys/firingsquad.cljc

```diff
@@ -28,15 +28,27 @@
   (let [lineup (common/build-lineup board stash)
         lineup-quantities (common/build-lineup-quantities lineup stash)
         raw-actions (common/build-deploy-actions player lineup-quantities 7)
-        actions (map deploy/build-deploy raw-actions)
+        actions (map deploy/deploy-action raw-actions)
+        board (assoc board :deploy-actions raw-actions)
         start-result (result/action-success board 0)]
     (reduce (partial common/do-actions player) start-result actions)))
 
+(defn build-history
+  "Takes the auto deploy action and adds the actual run deploy actions
+  on the history"
+  [result raw-actions]
+  (let[history (get-in result [:board :history])
```

Comment:
Isn't a space missing here? Would it make sense to add tests for this function? It would've probably caught this.